frontend/src/pages/Profile.tsx

```diff
@@ -0,0 +1,118 @@
+import { Descriptions, DescriptionsProps, Space, Typography } from 'antd';
+import { useEffect, useState } from 'react';
+import { fetchUserEmail } from '../services';
```

Comment:
See my later review comment for context
```suggestion
import { fetchUserEmail, fetchProfile } from '../services';
```